node/states.js

```diff
@@ -173,6 +191,41 @@ UnhealthyState.prototype.onRequestError = function onRequestError(err) {
     }
 };
 
-UnhealthyState.prototype.close = function close(callback) {
-    callback(null);
+// ## LockedHealthyState
```

Comment:
can we make this `// ## -- "` just so that we have all language's single-line commenting approaches in one?

Right, so please just make it a normal comment.